src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource.py

```diff
@@ -11,7 +11,7 @@
                                            ResourceGroupVCRTestBase, MOCKED_SUBSCRIPTION_ID)
 
 #pylint: disable=method-hidden
-class ResourceGroupScenarioTest(VCRTestBase):
+class ResourceGroupScenarioTest(VCRTestBase): # Not RG test base because it tests the actual deletion
```

Comment:
># Not RG test base because it tests the actual deletion [](start = 46, length = 55)

Can you add "actual deletion of a resource group" to make the comment clearer.